crypto/dsa/internal.h

```diff
@@ -40,6 +40,8 @@ struct dsa_st {
   CRYPTO_EX_DATA ex_data;
 };
 
+#define OPENSSL_DSA_MAX_MODULUS_BITS 10000
```

Comment:
Can this go in a public header file so the DSA generate comment can say something like "only supports generating up to |OPENSSL_DSA_MAX_MODULUS_BITS| bit keys"?